apps/darts-modernisation/darts-api/demo.yaml

```diff
@@ -8,4 +8,4 @@ spec:
   values:
     java:
       ingressHost: darts-api.demo.platform.hmcts.net
-      image: sdshmctspublic.azurecr.io/darts/api:prod-272ebc2-20230322092842 # {"$imagepolicy": "flux-system:darts-api-prod"}
```

Comment:
Do we not want to leave the image line in there for some of these files, so we can override the version that is deployed to a given environment at some point?

Reply:
can do, just demo for now ?